Buildings/Applications/DHC/EnergyTransferStations/Examples/CoolingDirectUncontrolled.mo

```diff
@@ -0,0 +1,157 @@
+    dpSup=6000,
+    dpRet=6000)
+    "Direct cooling energy transfer station"
+    annotation (Placement(transformation(extent={{-10,-60},{10,-40}})));
+
+  Buildings.Fluid.Sources.Boundary_pT souDis(
+    redeclare package Medium = Medium,
+    p(displayUnit="Pa") = 300000 + 800,
+    use_T_in=true,
+    T=280.15,
+    nPorts=1)
+    "District (primary) source"
+    annotation (Placement(transformation(extent={{-100,-20},{-80,0}})));
+
+  Buildings.Fluid.Sources.Boundary_pT sinDis(
+    redeclare package Medium = Medium,
+    p(displayUnit="Pa") = 300000,
+    T=289.15,
+    nPorts=1)
+    "District-side (primary) sink"
+    annotation (Placement(transformation(extent={{-100,-100},{-80,-80}})));
+
+  Buildings.Fluid.HeatExchangers.HeaterCooler_u loa(
+    redeclare package Medium = Medium,
+    allowFlowReversal=false,
+    m_flow_nominal=m_flow_nominal,
+    from_dp=false,
+    linearizeFlowResistance=true,
+    show_T=true,
+    energyDynamics=Modelica.Fluid.Types.Dynamics.SteadyStateInitial,
+    Q_flow_nominal=-1,
+    dp_nominal=100)
+    "Aggregate building cooling load"
+    annotation (Placement(transformation(extent={{80,-20},{100,0}})));
+
+  Modelica.Blocks.Sources.CombiTimeTable QCoo(
+    table=[0,-10E3; 6,-8E3; 6,-5E3; 12,-2E3; 18,-15E3; 24,-10E3],
+    timeScale=3600,
+    extrapolation=Modelica.Blocks.Types.Extrapolation.Periodic)
+    "Cooling demand"
+    annotation (Placement(transformation(extent={{-80,80},{-60,100}})));
+
+  Modelica.Blocks.Sources.Ramp ram(
+    height=1,
+    duration(displayUnit="h") = 18000,
+    startTime(displayUnit="h") = 3600)
+    "Ramp load from zero"
+    annotation (Placement(transformation(extent={{-80,40},{-60,60}})));
+
+  Modelica.Blocks.Math.Product pro
+    "Multiplyer to ramp load from zero"
+    annotation (Placement(transformation(extent={{40,46},{60,66}})));
+
+  Buildings.Fluid.Actuators.Valves.TwoWayEqualPercentage val(
+    redeclare package Medium = Medium,
+    allowFlowReversal=false,
+    m_flow_nominal=m_flow_nominal,
+    dpValve_nominal=50,
+    riseTime=10)
+    "In-building terminal control valve"
+    annotation (Placement(transformation(extent={{120,-20},{140,0}})));
+
+  Modelica.Blocks.Sources.RealExpression Q_flow_max(
+    y=-Q_flow_nominal)
+    "Maximum Q_flow"
+    annotation (Placement(transformation(extent={{-40,68},{-20,88}})));
+
+  Modelica.Blocks.Math.Division div
+    "Division"
+    annotation (Placement(transformation(extent={{0,74},{20,94}})));
+
+  Modelica.Blocks.Sources.Trapezoid tra(
+    amplitude=2,
+    rising(displayUnit="h") = 10800,
+    width(displayUnit="h") = 10800,
+    falling(displayUnit="h") = 10800,
+    period(displayUnit="h") = 43200,
+    offset=273.15 + 6)
+    "District supply temperature trapezoid signal"
+    annotation (Placement(transformation(extent={{-140,-16},{-120,4}})));
+
+equation
+  connect(tra.y, souDis.T_in)
+    annotation (Line(points={{-119,-6},{-102,-6}}, color={0,0,127}));
+  connect(QCoo.y[1], div.u1)
```

Comment:
`div.u1` should rather be connected to `pro.y` no?
Also since the denominator is constant, better use a gain than a division block.

Reply:
Correct on both accounts, thanks! Revised as suggested.